test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -81,40 +68,35 @@ public virtual DeploymentParameters GetDeploymentParameters()
 #else
 #error the target framework needs to be updated.
 #endif
+#if DEBUG
+                Configuration = "Debug",
```

Comment:
Interesting. Should the *test* always test both Debug and Release? Ultimately this is about the prospective user's app being in debug or release, no? If so, I think we should always test both ways. Or at least have some tests that test both.